### ISSUE TITLE
Using `skipMerge` and `nosuffix` has no effect and it just adds `.map` files

### ISSUE THREAD

**Issue by reflexdemon** [samaxes/minify-maven-plugin#123]
Hey,
I have a unique need. My application has been developed a while ago and I wanted to just minify my `js` and `css`. 
Here are my challenges,
1. Changing all my JSPs from `somefile.js` to `somefile.min.js` will be a huge project. In the interest of time I started using `<nosuffix>true</nosuffix>`.
2. At present I cannot do a merge of my `js` and `css` as the order in which the files are structured needs to be preserved as it is a legacy application.
Given this,
I used `skipMerge` and `nosuffix`  in my code and the execution-phase was set to package.

Here is my plugin config
```xml
<!-- minify js/css -->
					<!-- See http://samaxes.github.io/minify-maven-plugin/minify-mojo.html for documentation -->
					<plugin>
						<groupId>com.samaxes.maven</groupId>
						<artifactId>minify-maven-plugin</artifactId>
						<version>1.7.4</version>
						<executions>
							<execution>
								<id>default-minify</id>
								<phase>package</phase>
								<configuration>
									<charset>UTF-8</charset>
									<cssSourceDir>css</cssSourceDir>
									<cssSourceIncludes>
										<cssSourceInclude>**/*.css</cssSourceInclude>
									</cssSourceIncludes>
									<jsSourceDir>js</jsSourceDir>
									<jsSourceIncludes>
										<jsSourceInclude>**/*.js</jsSourceInclude>
									</jsSourceIncludes>
									<jsSourceExcludes>
										<jsSourceExclude>**/*.min.js</jsSourceExclude>
									</jsSourceExcludes>
									<jsEngine>CLOSURE</jsEngine>
									<closureLanguage>ECMASCRIPT5</closureLanguage>
									<skipMerge>true</skipMerge>
									<nosuffix>true</nosuffix>
									<skipMinify>false</skipMinify>
									<verbose>false</verbose>
									<closureCreateSourceMap>true</closureCreateSourceMap>
								</configuration>
								<goals>
									<goal>minify</goal>
								</goals>
							</execution>
						</executions>
					</plugin>
``


My war plugin looks like this.

```xml
<plugin>
						<groupId>org.apache.maven.plugins</groupId>
						<artifactId>maven-war-plugin</artifactId>
						<version>${mavenWarVersion}</version>
						<configuration>
							<attachClasses>true</attachClasses>
							<packagingExcludes>
								data/**/*.json,
								img/dummy,
								logback.xml,
								WEB-INF/lib/*.jar
							</packagingExcludes>
							<webXml>src/main/webapp/WEB-INF/web.xml</webXml>
							<warSourceIncludes>
								**/*.xml,
								**/*.html,
								**/*.css,
								**/*.map,
								**/*.eot,
								**/*.svg,
								**/*.ttf,
								**/*.woff,
								**/*.zip,
								**/*.hbs,
								**/*.jpg,
								**/*.png,
								**/*.swf,
								**/*.js,
								**/*.properties,
								**/*.jsp
							</warSourceIncludes>
						</configuration>
					</plugin>
```

I am able to see the `targer/${war-folder}` having the minified file but the packaged `war` does not contain any minified files. Is there a way to do the minification before the packaging and ship it along the war. My WAR project is published to the artifactory and other teams consumes the war as overlays. 

In short, the above configuration does not have any effect on to my `war` clients.
Is there anything I am missing? I am in need of help.

**Comment by Henrique-Lima-2016** [comment]
Solved:
https://github.com/samaxes/minify-maven-plugin/issues/50

**Comment by reflexdemon** [comment]
For the records I just want to update the answer to this thread,


I just added the WAR excluded inside the WAR plugin
```
 <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-war-plugin</artifactId>
                <version>2.2</version>
                <configuration>
                    ...
                    <warSourceExcludes>**/file-1.js,**/file-2.js,**/file1.css</warSourceExcludes>
                </configuration>
```

**Comment by samaxes** [comment]
I'm closing this issue since its a duplicate of #50.
